_posts/2023/2023-03-05-libreelec-nexus-11-0-0.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "LibreELEC (Nexus) 11.0"
+description: "LE 11.0 ... finally final"
+image: img/posts/icon-release-k20.jpg
+---
+
+The final stable version of LibreELEC 11 has been released, bringing Kodi (Nexus) v20.0.
+
+## NEWS
+
+With the new release cycle we add a Generic-Legacy image supporting nVidia cards, Chrome Browser add-on, and older hardware. We also reintroduce support for older Amlogic devices (S905, S905X/D, S912). If you use them make sure to read the paragraphs below.
+  
+LibreELEC 10.0 installs will not automatically update, but you can [manually update](https://wiki.libreelec.tv/support/update). Older LibreELEC installs must make a clean install due to the Python 3 changes since Kodi v19.
+
+## RASPBERRY PI
+
+- 50/60fps H264 HW decoding may need `force_turbo=1` or `core_freq_min=500` in config.txt to avoid AV-sync-issues/skipping
+- Kodi at RPi4 runs in 4096x2160 instead of 3840x2160 on 4k TVs after fresh installation. Configure Kodi as [described at the wiki](https://wiki.libreelec.tv/configuration/4k-hdr) and optionally add hdmi_enable_ekp60=1 to config.txt and enable HDMI UltraHD Deep Color in your TV's HDMI port configuration to get 4kp60 modes
```

Comment:
```suggestion
- Kodi at RPi4 runs in 4096x2160 instead of 3840x2160 on 4k TVs after fresh installation. Configure Kodi as [described at the wiki](https://wiki.libreelec.tv/configuration/4k-hdr) and optionally add hdmi_enable_4kp60=1 to config.txt and enable HDMI UltraHD Deep Color in your TV's HDMI port configuration to get 4kp60 modes
```